src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/PaymentAccountServiceImplTest.java

```diff
@@ -143,11 +143,16 @@ public void deleteUserAndPaymentAccountsFromUserAccountMapTest() {
 
     @Test
     public void deletePaymentAccountsFromOrganisationTest() {
-        when(organisationRepositoryMock.findByOrganisationIdentifier(any(String.class))).thenReturn(organisation);
+        Organisation organisationMock = mock(Organisation.class);
```

Comment:
prefix test to the test case

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/620/commits/b5448bbd468688f02bf1fd93ebc725364a468785